scripts/prepare_network.py

```diff
@@ -224,4 +224,9 @@ def average_every_nhours(n, offset):
     elif ll_type == 'c':
         set_line_cost_limit(n, factor, Nyears)
 
+    ln_config = snakemake.config["lines"]
+    lk_config = snakemake.config["links"]
+    n.lines.s_nom_max = ln_config.get("s_nom_max,", np.inf)
+    n.links.loc[n.links.carrier=='DC', "p_nom_max"] = lk_config.get("p_nom_max", np.inf)
+
```

Comment:
Don't you think it makes more sense to **clip** the maximum capacities to the upper limit. Here the limits which might be lower than `ln_config['s_nom_max']` for example are overwritten.

Reply:
True, did not think about it, because we never had limits on those.